.golangci.json

```diff
@@ -10,17 +10,37 @@
 			"gofmt"
 		]
 	},
+	"linters-settings": {
+		"govet": {
+			"check-shadowing": false
+		},
+		"gofmt": {
+			"simplify": false
```

Comment:
I have found these useful in the past, how much time does dropping these save?

Reply:
more than i care to admit. ill do a diff on the two but it's what weuse in rancher2, only reason i brought it forward. most everyone runs goimports locally any way i dont see why we need more than this but i could be talked out of it

Reply:
```
#with
level=info msg="Memory: 62 samples, avg is 325.6MB, max is 1366.2MB"
level=info msg="Execution took 6.0739442s"

#without
level=info msg="Memory: 62 samples, avg is 342.8MB, max is 1490.5MB"
level=info msg="Execution took 6.0010344s"
```

just changes max mem usage. execution time is basically the same. i dont care if we leave it but like i said, we disable it in rancher2 and dont seem to care over there

Reply:
I'd vote for leaving it, its just a few milliseconds difference and somehow actually takes MORE memory to turn it off?